src/platforms/react-native/sourcemaps.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Source Maps
+sidebar_order: 95
+redirect_from:
+  - /platforms/react-native/manual-setup/source maps/
+description: "Learn more about how to upload your source maps to Sentry."
+---
+
+To get unminified stack traces for JavaScript code, source maps must be generated and uploaded. The React Native SDK handles source maps _automatically_ for iOS with Xcode and Android with Gradle. However, if you use custom values for your release other than the version included with the build in XCode or Android Studio, you will need to manually upload source maps.
```

Comment:
clarifying q: while you can set dist and release, that interferes with the use of the automatic upload, correct? Asking because I think we'd want the easy path noted pretty clearly as source maps is often a source of confusion.

Suggest something like:
```suggestion
To get unminified stack traces for JavaScript code, source maps must be generated and uploaded. The React Native SDK handles source maps _automatically_ for iOS with Xcode and Android with Gradle, if you do not use custom values. 

However, if you use custom values for your release other than the version included with the build in XCode or Android Studio, the automatic source maps upload script will no longer work because it does not detect custom values. As a result, you'll need to manually upload source maps using these steps:
```

Reply:
Correct, setting dist and release instantly makes the auto sourcemaps useless and they'll be attributed to the default release and dist.